src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -499,6 +500,18 @@ static String validateConfig(Map<String, String> config) {
       }
     }
 
+    try {
+      SnowflakeSinkConnectorConfig.ProcessingGuarantee.of(
+          config.getOrDefault(PROCESSING_GUARANTEE, null));
```

Comment:
What will be `enum.of(null)`? Will it be allowed?

Reply:
Ignore, I checked the method. Looks good, but lets keep default to `AT_LEAST_ONCE`

Reply:
Yah, the enum.of(null) currently will return default AT_LEAST_ONCE. It also makes sense to throw exception when user provide an empty string, should i update the logic?